orderer/common/multichannel/registrar_test.go

```diff
@@ -1789,3 +1790,66 @@ func createLedgerAndChain(t *testing.T, r *Registrar, lf blockledger.Factory, b
 	r.CreateChain(channel)
 	require.NotNil(t, r.GetChain(channel))
 }
+
+func TestRegistrar_ConfigBlockOrPanic(t *testing.T) {
+	t.Run("Panics when ledger is empty", func(t *testing.T) {
+		tmpdir, err := ioutil.TempDir("", "file-ledger")
+		require.NoError(t, err)
+		defer os.RemoveAll(tmpdir)
+
+		_, l := newLedgerAndFactory(tmpdir, "testchannelid", nil)
+
+		assert.Panics(t, func() {
+			ConfigBlockOrPanic(l)
+		})
```

Comment:
- better to use `require` instead of `assert`
- Wherever possible better to use `PanicsWithValue` so you could test for the exact error message expected